### PR TITLE
Fire system

### PR DESCRIPTION
**Pull request by Yersen**
Base for the fire system

### CODE REVIEW

File: UtopiaCity/Common/Initializers/FireMessageInitializer.cs [hunk 1]
```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using UtopiaCity.Common.Interfaces;
+using UtopiaCity.Data;
+using UtopiaCity.Models.FireSystem;
+
```
Comment (crocomoth) [inline UtopiaCity/Common/Initializers/FireMessageInitializer.cs:8]: unnecessary empty line

File: UtopiaCity/Controllers/FireSystem/EmployeeManagementController.cs [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using UtopiaCity.Models.FireSystem.ManagerSystemTransportAndEmployees;
+using UtopiaCity.Services.FireSystem;
+
+namespace UtopiaCity.Controllers.FireSystem
+{
+
```
Comment (crocomoth) [inline UtopiaCity/Controllers/FireSystem/EmployeeManagementController.cs]: unnecessary empty line

File: UtopiaCity/Data/ApplicationDbContext.cs [hunk 3]
```diff
@@ -82,5 +84,9 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
         public DbSet<ArrivingPassenger> ArrivingPassengers { get; set; }
         public DbSet<ClinicVisit> ClinicVisit { get; set; }
         public DbSet<RealEstate> RealEstate { get; set; }
+        public DbSet<FireMessage> FireMessages { get; set; }
+        public DbSet<FireSafetyCheck> FireSafetyChecks { get; set; }
+        public DbSet<TransportManagement> TransportsManagements { get; set; }
```
Comment (crocomoth) [inline UtopiaCity/Data/ApplicationDbContext.cs]: naming is ambigious. Also not consistent

File: UtopiaCity/Models/FireSystem/ManagementSystemTransportAndEmployeess/Enums.cs [hunk 4]
```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace UtopiaCity.Models.FireSystem.ManagementSystemTransportAndEmployeess
+{
+    public class Enums
```
Comment (crocomoth) [inline UtopiaCity/Models/FireSystem/ManagementSystemTransportAndEmployeess/Enums.cs]: You don't really need this class, also enums can be stored in simple cs file(create -> new -> code file), without class

**Review by crocomoth — APPROVED** [review]

File: UtopiaCity/Data/ApplicationDbContext.cs (hunk 3 above)
Comment (Yersen) [reply UtopiaCity/Data/ApplicationDbContext.cs]: What namings do you suggest?

**Review by IvanSultanov — APPROVED** [review]

**Pull request closed without merging**